### ISSUE TITLE
Staging Review finding: Vague header for "programs" on school details page

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#105701]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Education Data Migration
**Product:** WEAMS Public
**Feature:** WEAMS Public

## Findings details
**VA.gov Experience Standard - issue:** [User has enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @briandeconinck (Accessibility)

### Description

The heading for the "Programs" section is broad and it's not immediately clear what can be found in that section of the page. Because this heading appears as a navigation item in the "On this page" component, and there are other sections of the page that have the word "program" in their heading, this heading should be clarified. Especially for users who are navigating the page by heading, the heading tag should be clear.

<details><summary>Link, screenshot or steps to recreate</summary>

to be added

</details>

## Recommended action

Update the heading text to clearly indicate what this section contains.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** [Page titles and section titles](https://design.va.gov/content-style-guide/page-titles-and-section-titles)
- **Context:** Navigation

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by skulka07** [comment]
Renamed "Programs" to title provided by business "Approved programs of study" & have updated it in all impacted locations.

![Image](https://github.com/user-attachments/assets/782be004-e72a-46f8-b02f-6e97fe3bf406)

![Image](https://github.com/user-attachments/assets/3993b4e5-584c-43c6-8e50-fd3dc51ef990)
